libraries/typescript/Microsoft.Bot.Builder.Adapters.WeChat/package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "wechat",
```

Comment:
>wechat [](start = 11, length = 6)

this should be the name of your package on npmjs.org.  This should probably be 
"botframework-wechat"